spec/feature_flagger/control_spec.rb

```diff
@@ -92,6 +108,11 @@ module FeatureFlagger
         expect(control.released?(key, 1)).to be_falsey
         expect(control.released_features_to_all).not_to include(key)
       end
+
+      it 'triggers notifer' do
```

Comment:
Acho que podemos melhorar as descrições aqui, podemos descrever o que de fato é esperado, tipo:
`it 'sends to notifier the unrelease to all event'` o que acha?